2017-08-02 13:30:24.8193|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:29.8171|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:34.8173|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:39.8182|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:44.8190|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:49.8351|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:54.8329|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:30:59.8337|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:31:04.8339|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:31:09.8369|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:31:14.8359|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:05.0590|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:09.6118|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:14.6177|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:19.6150|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:24.6286|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:29.6333|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:34.6371|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:39.6466|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:44.6470|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:49.6512|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:54.6517|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:33:59.6715|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:34:04.6769|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:34:09.6802|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:34:14.6937|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:34:19.7010|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:34:24.7181|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-02 13:34:29.7376|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Ошибка входа пользователя "NT AUTHORITY\LOCAL SERVICE".
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 614
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-03 12:20:32.2570|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Infrastructure.DbUpdateConcurrencyException: Store update, insert, or delete statement affected an unexpected number of rows (0). Entities may have been modified or deleted since entities were loaded. See http://go.microsoft.com/fwlink/?LinkId=472540 for information on understanding and handling optimistic concurrency exceptions. ---> System.Data.Entity.Core.OptimisticConcurrencyException: Store update, insert, or delete statement affected an unexpected number of rows (0). Entities may have been modified or deleted since entities were loaded. See http://go.microsoft.com/fwlink/?LinkId=472540 for information on understanding and handling optimistic concurrency exceptions.
   в System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.ValidateRowsAffected(Int64 rowsAffected, UpdateCommand source)
   в System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.Update()
   в System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.<Update>b__2(UpdateTranslator ut)
   в System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.Update[T](T noChangesResult, Func`2 updateFunction)
   в System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.Update()
   в System.Data.Entity.Core.Objects.ObjectContext.<SaveChangesToStore>b__35()
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   в System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   в System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   в System.Data.Entity.Internal.InternalContext.SaveChanges()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.SaveChanges()
   в System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   в System.Data.Entity.DbContext.SaveChanges()
   в DBFirstDAL.Repositories.CategoryRepository.AddOrUpdateFilterBrand() в C:\git\Pyramid\DBFirstDAL\Repositories\CategoryRepository.cs:строка 620
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 103
2017-08-03 13:04:41.9396|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.InvalidOperationException: Коллекция была изменена; невозможно выполнить операцию перечисления.
   в System.Collections.Generic.HashSet`1.Enumerator.MoveNext()
   в System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   в DBFirstDAL.Repositories.ProductRepository.UpdateAfterSavingOneC(PyramidFinalContext dbContext, Products dbObjext, Products model, Boolean exist) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 387
   в DBFirstDAL.Repositories.ProductRepository.AddOrUpdateFromOneC(Products product) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 333
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 100
2017-08-03 13:10:52.3722|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.InvalidOperationException: Коллекция была изменена; невозможно выполнить операцию перечисления.
   в System.Collections.Generic.HashSet`1.Enumerator.MoveNext()
   в System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   в DBFirstDAL.Repositories.ProductRepository.UpdateAfterSavingOneC(PyramidFinalContext dbContext, Products dbObjext, Products model, Boolean exist) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 387
   в DBFirstDAL.Repositories.ProductRepository.AddOrUpdateFromOneC(Products product) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 333
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 100
2017-08-03 13:18:07.0669|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.InvalidOperationException: Коллекция была изменена; невозможно выполнить операцию перечисления.
   в System.Collections.Generic.HashSet`1.Enumerator.MoveNext()
   в System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   в DBFirstDAL.Repositories.ProductRepository.UpdateAfterSavingOneC(PyramidFinalContext dbContext, Products dbObjext, Products model, Boolean exist) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 387
   в DBFirstDAL.Repositories.ProductRepository.AddOrUpdateFromOneC(Products product) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 333
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 100
2017-08-03 13:21:30.3364|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.InvalidOperationException: Коллекция была изменена; невозможно выполнить операцию перечисления.
   в System.Collections.Generic.HashSet`1.Enumerator.MoveNext()
   в System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   в DBFirstDAL.Repositories.ProductRepository.UpdateAfterSavingOneC(PyramidFinalContext dbContext, Products dbObjext, Products model, Boolean exist) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 387
   в DBFirstDAL.Repositories.ProductRepository.AddOrUpdateFromOneC(Products product) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 333
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic()
2017-08-09 16:16:39.8649|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Validation.DbEntityValidationException: Сбой при проверке одной или нескольких сущностей. Более подробные сведения см. в свойстве "EntityValidationErrors".
   в System.Data.Entity.Internal.InternalContext.SaveChanges()
   в System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   в System.Data.Entity.DbContext.SaveChanges()
   в DBFirstDAL.Repositories.ProductRepository.AddOrUpdateFromOneC(Products product) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 335
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 100
2017-08-09 16:18:43.8747|ERROR|WindowsServicePyramid.ServicePyramid|Ошибка при выгрузке информации на публичный сайт: System.Data.Entity.Validation.DbEntityValidationException: Сбой при проверке одной или нескольких сущностей. Более подробные сведения см. в свойстве "EntityValidationErrors".
   в System.Data.Entity.Internal.InternalContext.SaveChanges()
   в System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   в System.Data.Entity.DbContext.SaveChanges()
   в DBFirstDAL.Repositories.ProductRepository.AddOrUpdateFromOneC(Products product) в C:\git\Pyramid\DBFirstDAL\Repositories\ProductRepository.cs:строка 335
   в WindowsServicePyramid.ServicePyramid.ExecuteLogic() в C:\git\Pyramid\WindowsServicePyramid\ServicePyramid.cs:строка 100
